app/models/manageiq/providers/ansible_tower/configuration_manager/refresh_worker/collector.rb

```diff
@@ -0,0 +1,19 @@
+class ManageIQ::Providers::AnsibleTower::ConfigurationManager::RefreshWorker::Collector
+  def initialize(ems, target, _options = nil)
+    @ems        = ems
+    @connection = ems.connect
+    @target     = target
```

Comment:
so, yeah, target is not used yet. Not sure how targeted refresh would work. Maybe I drop it here